### PR TITLE
mel: new package

### PR DESCRIPTION
**Pull request by cristiano-campos**
Mel is a MPI implementation of Half-approximate Graph Matching.

@hyviquel

### CODE REVIEW

File: var/spack/repos/builtin/packages/mel/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+class Mel(MakefilePackage):
+    """MPI implementation of Half-approximate Graph Matching"""
+
+    homepage = "https://github.com/ECP-ExaGraph/mel"
+    git = "https://github.com/ECP-ExaGraph/mel.git"
+
+    maintainers = ["Cgbrl28"]
+
+    version("develop", branch="master")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/mel/package.py:17]: From a user perspective (and a general rule), the version name should match the branch.

File: var/spack/repos/builtin/packages/mel/package.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        if "+opt" in self.spec:
+            cxxflags.append("-O3")
+
+        targets.append("CXXFLAGS={0}".format(" ".join(cxxflags)))
+        targets.append("OPTFLAGS={0}".format(" ".join(ldflags)))
+        targets.append("CXX={0}".format(self.spec["mpi"].mpicxx))
+        return targets
+
+    def install(self, spec, prefix):
+
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/mel/package.py:42]: Suggest dropping this blank line.

**Review by tldahlgren — COMMENTED** [review]
Confirmed branch exists but have a couple of suggestions with the most important being to use the real branch name as the name of the version.

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Also, development seems to have stopped in the repo https://github.com/ECP-ExaGraph/mel

**Pull request closed without merging**